### ISSUE TITLE
[New Rule - TS2] Multiple tabs on single element

### ISSUE THREAD

**Issue by marcjohlic** [IBMa/equal-access#852]
For 2.4.3 Focus Order: Need a rule for when the focus (appears) not to move when the Tab key is hit more than once.

See the [Slack thread](https://ibm-studios.slack.com/archives/GSGN38DQE/p1650902482592629) for more information.

@mbgower can you please give a better description of the intent behind this idea?

**Comment by mbgower** [comment]
Yes, something like this <div tabindex="0"><button>I will get focus twice</button></div>

**Comment by mbgower** [comment]
BTW, there is a big [discussion going on in an issue/PR](https://github.com/w3c/wcag/issues/1572) about this exact scenario and the degree to which it IS a failure of Focus Order

**Comment by MHoov** [comment]
@drjoho @marcjohlic this issue needs to be assigned.

**Comment by shunguoy** [comment]
https://ibm.ent.box.com/notes/955840621385

**Comment by marcjohlic** [comment]
@philljenkins will you be using this issue for the Help on this rule, or is there (will there be) a separate Help issue for TS2 Multiple tabs on single element Help?

**Comment by philljenkins** [comment]
@marcjohlic there was a separate issue opened for the Help and linked to this issue, #889

**Comment by philljenkins** [comment]
@shunguoy 
Change ruleID to `element_tabbable_role_valid`  
and messages to:
`"fail_invalid_role": The tabbable element's role '{0}' is not a valid widget role`
`"group": A tabbable element must have a valid widget role`

**Comment by philljenkins** [comment]
also change the ruleset mappings from IBM and WCAG20 & WCAG21 from 2.4.3 to requirement 4.1.2:
`"num": ["4.1.2"],`

**Comment by philljenkins** [comment]
Help changes are implemented in branch dev-852
